csrc/ops/indexing.h

```diff
@@ -83,4 +83,10 @@ NVF_API TensorView* takeAlongAxis(
     TensorView* index,
     int64_t dim);
 
+NVF_API TensorView* groupedBlockSfLayout(
+    TensorView* input,
+    TensorView* expert_offsets,
```

Comment:
Let's not use application-specific naming since the function name is not specific to the application (MoE).